### ISSUE TITLE
Wrong WMS link under gmd:MD_BrowseGraphic XML tag using diva2xml

### ISSUE THREAD

**Issue by meliezer** [gher-ulg/DIVAnd.jl#12]
I see an error also in the link inside the test.xml file:
         <gmd:graphicOverview xmlns:srv="http://www.isotc211.org/2005/srv">
            <gmd:MD_BrowseGraphic>
               <gmd:fileName>
                  <gco:CharacterString>http://ec.oceanbrowser.net/emodnet/Python/web/wms?service=WMS&amp;request=GetMap&amp;version=1.3.0&amp;crs=CRS%3A84&amp;bbox=-42.0%2C24.9%2C-0.099998474%2C48.0&amp;decorated=true&amp;format=image%2Fpng&amp;layers=Atlantic%20Ocean%2FWater_body_chlorophyll-a.4Danl.nc%2AWater_body_chlorophyll-a&amp;styles=vmin%3A2.9031277%2Bvmax%3A7.8576584&amp;transparent=true&amp;height=500&amp;width=800</gco:CharacterString>
               </gmd:fileName>
               <gmd:fileDescription>
                  <gco:CharacterString>large_thumbnail</gco:CharacterString>
               </gmd:fileDescription>
               <gmd:fileType>
                  <gco:CharacterString>png</gco:CharacterString>
               </gmd:fileType>
            </gmd:MD_BrowseGraphic>
         </gmd:graphicOverview>

**Comment by Alexander-Barth** [comment]
Can you show me the divadoxml command that you used, in particular the value of the filepath parameter?

**Comment by meliezer** [comment]
using DIVAnd;
  files = [
       	"Winter (January-March) - 6-years running averages/Water_body_chlorophyll-a.4Danl.nc",
       	"Spring (April-June) - 6-years running averages/Water_body_chlorophyll-a.4Danl.nc",
       	"Summer (July-September) - 6-years running averages/Water_body_chlorophyll-a.4Danl.nc",
       	"Autumn (October-December) - 6-years running averages/Water_body_chlorophyll-a.4Danl.nc"
       	];
  DIVAnd.divadoxml(files,"Water_body_chlorophyll-a","EMODNET-chemistry","export.zip","test.xml";
    ignore_errors = true,
    additionalvars = Dict("abstract" => "Here goes the abstract"),
    WMSlayername = ["winter","spring","summer","autumn"]
)

**Comment by Alexander-Barth** [comment]
There is a problem with the basin name. In OceanBrowser, we call the domain "Atlantic Sea", while in vocab C19 it is "Atlantic Ocean". 

Another issue is that the preview defaults to the first time instance which is pretty empty:

http://ec.oceanbrowser.net/emodnet/Python/web/wms?service=WMS&request=GetMap&version=1.3.0&crs=CRS%3A84&bbox=-42.0%2C24.9%2C-0.099998474%2C48.0&decorated=true&format=image%2Fpng&layers=Atlantic%20Sea%2FWinter%20%28January-March%29%20-%206-years%20running%20averages%2FWater_body_chlorophyll-a.4Danl.nc%2AWater_body_chlorophyll-a_L1&styles=vmin%3A2.9031277%2Bvmax%3A7.6490226&transparent=true&height=500&width=800

**Comment by Alexander-Barth** [comment]
I have updated the master branch. It should now give you the image URL above.

**Comment by Alexander-Barth** [comment]
Here is my full script. You see below I had to replace "Atlantic Ocean" by "Atlantic Sea".
I think the best option would be to use "Atlantic Ocean" but this would require to change all old metadata records in Sextant. I don't know if now is the best time to do this.

```julia
using DIVAnd
files = [
         "Winter (January-March) - 6-years running averages/Water_body_chlorophyll-a.4Danl.nc",
         "Spring (April-June) - 6-years running averages/Water_body_chlorophyll-a.4Danl.nc",
         "Summer (July-September) - 6-years running averages/Water_body_chlorophyll-a.4Danl.nc",
         "Autumn (October-December) - 6-years running averages/Water_body_chlorophyll-a.4Danl.nc"
];

DIVAnd.divadoxml(files,"Water_body_chlorophyll-a","EMODNET-chemistry","export.csv","test.xml";
                 ignore_errors = true,
                 additionalvars = Dict("abstract" => "Here goes the abstract"),
                 WMSlayername = ["winter","spring","summer","autumn"]
)

xmlfile = readstring("test.xml");
xmlfile = replace(xmlfile,"Atlantic Ocean" => "Atlantic Sea");
xmlfile = replace(xmlfile,"Atlantic%20Ocean" => "Atlantic%20Sea");
write("test2.xml",xmlfile);
```

**Comment by Alexander-Barth** [comment]
But changing manually the year, vmin, vmax and the basemap, we can get a better preview. 


http://ec.oceanbrowser.net/emodnet/Python/web/wms?service=WMS&request=GetMap&version=1.3.0&crs=CRS%3A84&bbox=-42.0%2C24.9%2C-0.099998474%2C48.0&decorated=true&format=image%2Fpng&layers=Atlantic%20Sea%2FWinter%20(January-March)%20-%206-years%20running%20averages%2FWater_body_chlorophyll-a.4Danl.nc*Water_body_chlorophyll-a_L1&styles=vmin%3A0%2Bvmax%3A1.8&transparent=true&height=500&width=800&time=2010&basemap=shadedrelief

**Comment by meliezer** [comment]
Thank you, but actually the more specific C19 term as found also in the Sextant records per season created by Julie is "Northeast Atlantic Ocean (40W)". I also see it in the netcdf: area_keywords = "Atlantic Ocean";
If not all current Atlantic Sextant records follow this C19 (We should verify if it's correct also for the old DIVA maps), then there is no problem to modify them.
We have another ocean case in the new products: Arctic Ocean.
As for the need for manual adjustment for having a better preview, you may add it to the documentation. It really improves the preview!

**Comment by meliezer** [comment]
I've tried and now I do sea "Sea" in the WMS link, however, I would need a clickable link while I get
http:&#x2F;&#x2F;ec.oceanbrowser.net&#x2F;emodnet&#x2F;Python&#x2F;web&#x2F;wms?service=WMS&amp;request=GetMap&amp;version=1.3.0&amp;crs=CRS%3A84&amp;bbox=-42.0%2C24.9%2C-0.099998474%2C48.0&amp;decorated=true&amp;format=image%2Fpng&amp;layers=Atlantic%20Sea%2FWinter%20%28January-March%29%20-%206-years%20running%20averages%2FWater_body_chlorophyll-a.4Danl.nc%2AWater_body_chlorophyll-a_L1&amp;styles=vmin%3A2.9031277%2Bvmax%3A7.6490226&amp;transparent=true&amp;height=500&amp;width=800
After decoding it using online tool I still fail to get the right link. It's not so important fixing it, since in any case I prefer adjust it manually for having a better preview, as you've written. It would be enough to know how to convert the link the script provides, even though I can follow your template for any product and compose the URL by myself.
Only a minor issue. I'm please you've identifed the Sea and Ocean problem. Thanks.

**Comment by Alexander-Barth** [comment]
The image link that you provided give a valid image for me. What you do mean by click-able link? Do you mean a change in the Sextant user-interface that makes this URL click-able? If yes, you might contact the Sextant group for this. divadoxml just provides the data (i.e. the link URL), the presentation is done in Sextant.

**Comment by meliezer** [comment]
No, I simply mean that the link is wrong. I've pasted it in two different browsers (latest versions).
http:&#x2F;&#x2F;ec.oceanbrowser.net&#x2F;emodnet&#x2F;Python&#x2F;web&#x2F;wms?service=WMS&amp;request=GetMap&amp;version=1.3.0&amp;crs=CRS%3A84&amp;bbox=-42.0%2C24.9%2C-0.099998474%2C48.0&amp;decorated=true&amp;format=image%2Fpng&amp;layers=Atlantic%20Sea%2FWinter%20%28January-March%29%20-%206-years%20running%20averages%2FWater_body_chlorophyll-a.4Danl.nc%2AWater_body_chlorophyll-a_L1&amp;styles=vmin%3A2.9031277%2Bvmax%3A7.6490226&amp;transparent=true&amp;height=500&amp;width=800

**Comment by ctroupin** [comment]
I've tried the link and get this image, is it ok?
![wms](https://user-images.githubusercontent.com/11868914/45092627-5cda3000-b116-11e8-909c-4996ce06ed0f.png)
